### ISSUE TITLE
set up  raspberry pi  only as  an access point 

### ISSUE THREAD

**Issue by wwj718** [resin-io/resin-wifi-connect#195]
Hi
The project is awesome！
Can I use resin-wifi-connect to set up  raspberry pi  only as  an access point ？
have any command line options to do it？

**Comment by majorz** [comment]
At the moment we do not have the option to setup the application in access point mode only without running the HTTP server that provides the user interface.

Can you please let me know more about your use case?

**Comment by wwj718** [comment]
Thanks for your reply.
In my case , I use raspberrypi as master note(control center) , two esp8266(micropython)  will connect it

**Comment by majorz** [comment]
Last week we started discussing the ability to accommodate such use cases, where there is a second application of control center type that can take advantage of the access point our application provides. I am not sure when this will be ready, as we are in the process of figuring out the details, but I will update you once we have more progress on this.

**Comment by wwj718** [comment]
That's great!
Thanks a lot

**Comment by CarlosLabrado** [comment]
Hello, guys, I am really interested in this feature as well, I have a Flask server running in port 80 displaying some settings for the device and I just need any other device to be able to connect to the AP and get a valid IP.

**Comment by wwj718** [comment]
hi @majorz   How is it going?

**Comment by majorz** [comment]
@CarlosLabrado @wwj718 please check the current progress with a demo application at https://github.com/resin-io-projects/resin-wifi-connect-api

It demonstrates how to run WiFi Connect on a different port than 80 and to have a different app that runs on port 80.

**Comment by CarlosLabrado** [comment]
Yeah, I ended up doing it that way, but it would be great to be able to assign your own IP and not run a separate container.
